platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/AnonClaimsHandlerExt.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.ui.admin.api;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AnonClaimsHandlerExt {
+    public static final String PROFILE_NAME_KEY = "profileName";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AnonClaimsHandlerExt.class);
+
+    private Map<String, Object> claimsProfiles;
+
+    private List<String> availableClaims;
+
+    //holds it name of claims that if modified would cause anonymous login failures
```

Comment:
"holds it name"? If 'it' is an acronym, maybe that should be spelled out, otherwise this is probably a typo?